OpenStreetMapViewer/build.gradle

```diff
@@ -1,33 +1,10 @@
 description = 'OpenMap in the Play Store, the example app for using osmdroid'
 
 apply plugin: 'com.android.application'
-apply from: "https://raw.githubusercontent.com/gradle-fury/gradle-fury/v1.1.4/gradle/android-support.gradle"
-
-// Attention! HACK!
-// In https://raw.githubusercontent.com/gradle-fury/gradle-fury/v1.1.4/gradle/android-support.gradle
-// we have a hack to prevent builds from failing due to missing signing config. This is done by
-// removing the signing task from the build if no keystore/pass is provided.
-//
-// At some point the signing task has been renamed from "validateReleaseSigning" to
-// "validateSigningRelease", thus obviously breaking the build again.
-//
-// The following is just a copy of the relevant code from android-support.gradle adapted to the new
-// name. This should be cleaned up in the future!
-afterEvaluate { project ->
-    tasks.each { task ->
-        if (isAndroidApplication) {
-            if ((task.name.toLowerCase().startsWith("assemble") && task.name.toLowerCase().contains('release'))
-                    || (task.name.toLowerCase().startsWith("bundle") && task.name.toLowerCase().contains('release'))
-                    || (task.name.toLowerCase().startsWith("package") && task.name.toLowerCase().contains('release'))
-                    || (task.name.equals("validateSigningRelease"))) {
-
-                task.setEnabled(canSign)
-            }
-        }
-    }
-}
 
 android {
+    compileSdkVersion findProperty('android.compileSdkVersion').toInteger()
```

Comment:
There are much easier ways to to that

Reply:
Such as?

Reply:
```
buildscript {
    ext {
        sdkCompileVersion = 29
        ....
```
```
android {
    compileSdkVersion sdkCompileVersion
```

No need having `gradle.properties`

Reply:
That doesn't seem to be a core requirement to updating the build tools used, so it's probably better left for a different PR.